virtcontainers/kata_agent.go

```diff
@@ -1003,6 +1056,14 @@ func (k *kataAgent) createContainer(sandbox *Sandbox, c *Container) (p *Process,
 
 	k.handleShm(grpcSpec, sandbox)
 
+	caps := sandbox.hypervisor.capabilities()
+	if !caps.is9pSupported() {
+		// 9p is not supported, files must be copied
+		if err := k.copyFiles(c.mounts, newMounts); err != nil {
```

Comment:
@egernst does this mean even files coming in from tmpfs, secrets all of them get copied? What happens to volumes.